docs/leap_sapi/sapi_rest.rst

```diff
@@ -1607,9 +1607,8 @@ to get a subset of solver fields.
             :skipif: test_api_token_set == False
 
             >>> params = {"filter": "none,+identity,+status,+avg_load"}
-            >>> r = requests.get(f"{SAPI_HOME}/solvers/remote/", params=params,
-            ...                  headers={'X-Auth-Token': SAPI_TOKEN,
-            ...                  'Accept': 'application/vnd.dwave.sapi.solver-definition-list+json; version=3'})
+            >>> r = session.get(f"{SAPI_HOME}/solvers/remote/", params=params,
+            ...                  headers={'Accept': 'application/vnd.dwave.sapi.solver-definition-list+json; version=3'})
```

Comment:
This is one of only two places where you are setting the `Accept` header inside the GET. In ten places we prefer the `>>> session.headers['Accept'] =...` construct, because it's easier for users to parse.
Why not here too?